packages/pnpm/test/list.ts

```diff
@@ -221,7 +221,7 @@ test('list on a project with skipped optional dependencies', async (t: tape.Test
   }
 
   {
-    const result = execPnpmSync('list', '--depth', '10', 'not-compatible-with-any-os')
+    const result = execPnpmSync('why', 'not-compatible-with-any-os')
```

Comment:
we probably also need to test the `list --depth 10` command as well, so please keep that and add a new assertion for `why`